src/common/utils/api.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+export const fetchProducts = async (limit, pageNumber, minPrice, maxPrice, categories, rating) => {
+  /* eslint-disable-next-line max-len */
```

Comment:
normally, you should url to a function that takes parameters and builds the string, the max-length rule was introduced because at the later stages it will become a maintainability pain-point.

However, we can ignore this as a one-off for now.

Reply:
Understandable, marking as resolved.